main/src/ca/uwaterloo/flix/runtime/shell/Command.scala

```diff
@@ -104,90 +109,51 @@ object Command {
     * Parses the given `input` into a command.
     */
   def parse(input: String)(implicit terminal: Terminal): Command = {
-    //
-    // Eof
-    //
```

Comment:
Surely at least this one is worth keeping :)

Reply:
Alright. Added it back.

Reply:
I made changes, but you don't have to approve those? Or is it because you did not use "changes requested"?